src/core/Surface/Surface.tsx

```diff
@@ -0,0 +1,66 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import { CommonProps, useTheme } from '../utils';
+import '@itwin/itwinui-css/css/surface.css';
+
+/**
+ * Helper function that returns one of the preset surface elevation values.
+ */
+const getSurfaceElevationValue = (elevation: SurfaceProps['elevation']) => {
+  if (!elevation) {
+    return '';
+  }
+
+  switch (elevation) {
+    case 4:
+      return '0 1px 10px rgba(0, 0, 0, 0.25)';
+    case 8:
+      return '0 3px 14px rgba(0, 0, 0, 0.25)';
+    case 16:
+      return '0 6px 30px rgba(0, 0, 0, 0.25)';
+    case 24:
+      return '0 9px 46px rgba(0, 0, 0, 0.25)';
+    default:
+      return '';
+  }
+};
+
+export type SurfaceProps = {
+  /**
+   * Sets the size of the box shadow.
+   * @default 2
+   */
+  elevation?: 2 | 4 | 8 | 16 | 24;
+  /**
+   * Content in the surface.
+   */
+  children: React.ReactNode;
+} & CommonProps;
+
+/**
+ * The surface container allows content to appear elevated through the use of a drop shadow
+ * @example
+ * <Surface>Surface Content</Surface>
+ * <Surface elevation={2}>Surface Content</Surface>
+ */
+export const Surface = (props: SurfaceProps) => {
+  const { elevation, className, style, children, ...rest } = props;
+  useTheme();
+
+  const _style = {
+    '--iui-surface-elevation': getSurfaceElevationValue(elevation),
```

Comment:
if you check other compoonents like `Badge` or `ColorBuilder` we do check if css supports variables. Not sure, if we want to continue doing that as we move away from IE, though:/
Would need to gather more opinions on this one

Reply:
Added IE support just in case